runner/src/server/services/uploadService.ts

```diff
@@ -224,4 +242,32 @@ export class UploadService {
     const promises = paths.map((path) => get<string>(path, {}));
     return Promise.all(promises);
   }
+
+  async uploadFilesS3(files) {
+    let response = new Array();
+
+    for (const file of files) {
+      const uploadParams = {
+        Bucket: bucketName,
+        Body: file,
+        Key: file.hapi.filename,
+      };
+
+      await s3
+        .upload(uploadParams)
+        .promise()
+        .then(function (data) {
+          console.log(data);
```

Comment:
Guess this is left in for debugging? Can we either log something using the logger, or remove this line?

Reply:
Well spotted, removed this